### PR TITLE
PKGBUILD: remove architecture specific stuff since we build with 32-bit ...

### PR DESCRIPTION
**Pull request by matthiaskrgr**
...support by default

### CODE REVIEW

**Comment by karolherbst** [comment]
will this break on 32bit systems?

**Comment by matthiaskrgr** [comment]
well I hope not since we have 32-bit enabled by default.

**Comment by karolherbst** [comment]
I am asking because of "binutils-multilib". We don't need a multilib compiler on 32bit systems

File: distro/archlinux/git/PKGBUILD [hunk 1]
```diff
@@ -20,12 +20,6 @@ _gitroot=git://github.com/lodle/Desurium.git
 _gitname=Desurium
 
 
-if [ "${CARCH}" = "x86_64" ]; then
-    depends+=('binutils-multilib')
-  else  # 32 bit
-    depends+=('binutils')
-fi
-
```
Comment (matthiaskrgr) [inline distro/archlinux/git/PKGBUILD:28]: mh, well,I could re-add (keep) this and

File: distro/archlinux/git/PKGBUILD [hunk 2]
```diff
@@ -53,22 +47,12 @@ build() {
...
     -DRUNTIME_LIBDIR=/usr/lib/
     -DBUILD_CEF=ON"
-  else  # 32 bit
-    CMAKE_ARGS="
-    -DCMAKE_INSTALL_PREFIX=/usr
-    -DBINDIR=/usr/bin
-    -DDATADIR=/usr/share/
-    -DRUNTIME_LIBDIR=/usr/lib/
-    -DBUILD_CEF=ON"
-  fi
```
Comment (matthiaskrgr) [inline distro/archlinux/git/PKGBUILD:71]: just remove this, would that be ok?

**Comment by karolherbst** [comment]
yeah, would be fine

File: distro/archlinux/git/PKGBUILD (hunk 1 above)
Comment (makson96) [inline distro/archlinux/git/PKGBUILD:28]: The same thing was just yesterday removed from install_deps.sh and I don't really know why...
https://github.com/lodle/Desurium/commit/e20b00597f77bf8db6df65e20045d27f5f9cb72a

Comment (karolherbst) [inline distro/archlinux/git/PKGBUILD:28]: yeah right, my mistake. I noticed this error too late. Thx for report

File: distro/archlinux/release/cef/PKGBUILD [hunk 3]
```diff
@@ -46,11 +46,12 @@ build() {
...
-    -DBINDIR=/usr/bin
-    -DDATADIR=/usr/share/
-    -DRUNTIME_LIBDIR=/usr/lib/
-    -DBUILD_ONLY_CEF=ON"
+   -DCMAKE_INSTALL_PREFIX=/usr
+   -DBINDIR=/usr/bin
+   -DDATADIR=/usr/share/
+   -DRUNTIME_LIBDIR=/usr/lib/
+   -DBUILD_ONLY_CEF=ON"
+
```
Comment (karolherbst) [inline distro/archlinux/release/cef/PKGBUILD:55]: consistent whitespaces?

File: distro/archlinux/release/desurium/PKGBUILD [hunk 4]
```diff
@@ -52,22 +52,13 @@ build() {
...
-    -DRUNTIME_LIBDIR=/usr/lib/
-    -DBUILD_CEF=OFF"
-  fi
+
+  CMAKE_ARGS="
+   -DCMAKE_INSTALL_PREFIX=/usr
+   -DBINDIR=/usr/bin
+   -DDATADIR=/usr/share/
+   -DRUNTIME_LIBDIR=/usr/lib/
+   -DBUILD_CEF=OFF"
```
Comment (karolherbst) [inline distro/archlinux/release/desurium/PKGBUILD:62]: here too

Comment (matthiaskrgr) [inline distro/archlinux/release/desurium/PKGBUILD:62]: - It has allways been that way
- It's more readable that way
- You also added some lines to the lis without changing the indent

Comment (karolherbst) [inline distro/archlinux/release/desurium/PKGBUILD:62]: why 3 spaces and not 4? (it was 4 before, now it is only 3)

Comment (matthiaskrgr) [inline distro/archlinux/release/desurium/PKGBUILD:62]: oh I see it now :|

**Pull request closed without merging**